rules/linux/process_creation/proc_creation_lnx_userdel.yml

```diff
@@ -0,0 +1,23 @@
+title: Delete User or Group Using Userdel or Groupdel
+id: 8a46f16c-8c4c-82d1-b121-0fdd3ba70a84
+status: experimental
+description: Detects usage of the "userdel" or "groupdel" binary to delete user or delete group
+references:
+    - https://cve.mitre.org/cgi-bin/cvename.cgi?name=2019-19882
```

Comment:
Why this ref ?

Reply:
I see this ref related to process groupdel and userdel

Reply:
This CVE is for shadow 4.8 and misconfigured setuid.

I am the operator who receives this alert.
I read the references, how I make the link between this CVE and this basic detection on a process execution.

If it is a privileged escalation that we want to detect, there is some information missing in the detection.
This is more a "simple"  [T1531 Account Access Removal ](https://attack.mitre.org/techniques/T1531/)

Reply:
Ok, thank you